integrations/wiz/.port/resources/port-app-config.yaml

```diff
@@ -16,7 +16,9 @@ resources:
             description: .description
   - kind: issue
     selector:
-      query: 'true'
+      query: "true"
+      maxPages: 20
```

Comment:
I'm curious why the default is 20. Is this the maximum number of pagination request that can be made or the number of issues that can be returned?

Reply:
This is the number of pages. Each page fetch returns 100 items. So by default 2000 items will be returned

Reply:
Gotcha. For other integrations that brings in issues such as Snyk, SonarQube, Sentry etc, it is possible to ingest any number of entities without limitation. is the 2000 the limit recommended by Wiz or something that was agreed with the customers who are using the integration?